src/sage/geometry/hyperplane_arrangement/arrangement.py

```diff
@@ -241,19 +247,20 @@
 :meth:`~HyperplaneArrangementElement.intersection_poset` of the
 arrangement and `\mu` is the Möbius function of `P`::
 
+    sage: # long time
     sage: a = hyperplane_arrangements.semiorder(5)
-    sage: a.characteristic_polynomial()               # long time (about a second on Core i7)
+    sage: a.characteristic_polynomial()               # about a second on Core i7
     x^5 - 20*x^4 + 180*x^3 - 790*x^2 + 1380*x
```

Comment:
This is very nice. Is this automatically done?

Reply:
No, I did this manually

Reply:
I see that the automatic result is
```diff
+    sage: # long time
     sage: a = hyperplane_arrangements.semiorder(5)
     sage: a.characteristic_polynomial()               # long time (about a second on Core i7)
     x^5 - 20*x^4 + 180*x^3 - 790*x^2 + 1380*x
-    sage: a.poincare_polynomial()                     # long time
+    sage: a.poincare_polynomial()
...
```
It seems that not touching the tag followed by a parenthesized remark results in somewhat lamed tags (the tag is still there though it is obviously redundant). 

Do you plan to enhance `sage -fixdoctests` to handle this case adequately as it was done manually?

Reply:
I think this case is rare enough that it can be left to manual fixing

Reply:
I imagine that It would be difficult to handle all cases perfectly though.

A compromise would be to remove only the tags but to leave the parenthesized remarks intact (for human intervention).

Reply:
> I think this case is rare enough that it can be left to manual fixing

OK.